crouton.php

```diff
@@ -896,7 +896,8 @@ function admin_options_page() {
 										<?php $area_parent = $area_data[2]; ?>
 										<?php $area_parent_name = $area_data[3]; ?>
 										<?php $option_description = $area_name . " (" . $area_id . ") " . $area_parent_name . " (" . $area_parent . ")" ?></option>
-										<?php if ($unique_area == $this->options['service_body_1']) { ?>
+										<?php $is_data = explode(',',esc_html($this->options['service_body_1'])); ?>
```

Comment:
yes so instead of comparing all of the unique_area info, this is just comparing the area ids, which are numbers.

Reply:
why doesn't this occur in bread?

Reply:
because bread does this, compares by area id
https://github.com/radius314/bread/blob/8c2882ef3fdc9bcd67c060f194c3aa9af10ae08d/partials/_bmlt_server_setup.php#L40-L43